index.js

```diff
@@ -0,0 +1,111 @@
+var net = require('net');
+var tls = require('tls');
+var http = require('http');
+var https = require('https');
+var ws = require('ws');
+var _ = require('underscore');
```

Comment:
I'm kind of against using toolbelts in tiny libraries, like this one, we can do without it :)